tensorflow/core/ops/math_ops.cc

```diff
@@ -392,7 +392,7 @@ Returns x * y element-wise.
 REGISTER_OP("Div").BINARY_MORE().SetShapeFn(
     shape_inference::BroadcastBinaryOpShapeFn);
 
-REGISTER_OP("UnsafeDiv")
+REGISTER_OP("DivNoNan")
     .BINARY_MORE()
```

Comment:
I forget to replace BINARY_MORE by float32, float64 only. Push the change commit later.